vaadin-grid-flow-parent/vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.js

```diff
@@ -906,7 +906,11 @@ import { isFocusable } from '@vaadin/grid/src/vaadin-grid-active-item-mixin.js';
           grid.$server.confirmParentUpdate(id, parentKey);
 
           if (!grid.loading) {
-            grid.__updateVisibleRows();
+            grid.__confirmParentUpdateDebouncer = Debouncer.debounce(
+              grid.__confirmParentUpdateDebouncer,
+              microTask,
+              () => grid.__updateVisibleRows()
```

Comment:
Why do we need to update visible rows? Isn't it already covered by _debouncerApplyCachedData in the web component?

Reply:
I considered the option of removing the call altogether. But in order to keep focus on performance enhancements within the ongoing project, I'll just assume that the call was originally added here for a reason. Once we have the connector fully covered with tests, it's going to be less risky to remove things we assume to be no longer necessary.

Reply:
OK, makes sense.